src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Workspaces/SetAzureOperationalInsightsWorkspaceCommand.cs

```diff
@@ -38,13 +38,18 @@ public class SetAzureOperationalInsightsWorkspaceCommand : OperationalInsightsBa
 
         [Parameter(Position = 3, Mandatory = false, ValueFromPipelineByPropertyName = true,
             HelpMessage = "The service tier of the workspace.")]
-        [ValidateSet("free", "standard", "premium", IgnoreCase = true)]
+        [ValidateSet("free", "standard", "premium", "pernode", "standalone", IgnoreCase = true)]
         public string Sku { get; set; }
 
         [Parameter(Position = 4, Mandatory = false, ValueFromPipelineByPropertyName = true,
             HelpMessage = "The resource tags for the workspace.")]
         public Hashtable Tags { get; set; }
 
+        [Parameter(Position = 5, Mandatory = false, ValueFromPipelineByPropertyName = true,
```

Comment:
Please remove the position property from this parameter.  > 4 positional parameters is too many.